src/SchemaUtils.elm

```diff
@@ -706,6 +738,32 @@ typeToEncoder qualify type_ =
         Common.String ->
             CliMonad.succeed Gen.Json.Encode.call_.string
 
+        Common.DateTime ->
+            CliMonad.succeed
+                (\instant ->
+                    Gen.Rfc3339.make_.dateTimeOffset
+                        (Elm.record
+                            [ ( "instant", instant )
+                            , ( "offset"
+                              , Elm.record
+                                    [ ( "hour", Elm.int 0 )
+                                    , ( "minute", Elm.int 0 )
+                                    ]
+                              )
+                            ]
+                        )
+                        |> Gen.Rfc3339.toString
```

Comment:
I don't love this

Reply:
I could see it being an issue if someone wanted to send date-times to their server with a non-zero offset. To me that seems like a silly thing to do, but people love to do silly things.

My counter argument is we should guide people towards better solutions and a zero offset is better IMO.

Reply:
Yeah, if people want to we can... ... ...add another flag :sweat_smile:
(a flag for "accept the record instead of the `Time.Posix` as argument)